windows.sh

```diff
@@ -33,6 +33,7 @@ fi
 if [ ! -f /usr/bin/ansible ]; then
   echo "Installing pip..."
   sudo apt-get -y install python-pip
+  sudo apt-get -y update
```

Comment:
I am testing from scratch to verify it's working correctly. I just made that change so its before.